bbp/default.nix

```diff
@@ -232,6 +238,18 @@ let
 
         };
 
+        pynrrd = callPackage ./nse/pynrrd {};
+        equation = callPackage ./nse/equation {};
+        numpy_1_13_1 = callPackage ./nse/numpy/1.13.1.nix {};
```

Comment:
About numpy:

Normally extension or update of standard packages ( non-BBP ones ) goes under patches. There is also a section for python related ones.

But like this is temporary and should disappear with update 17.03, I this we can keep that